jar/jar.go

```diff
@@ -299,6 +307,14 @@ func (c *checker) checkJAR(r *zip.Reader, depth int, size int64) error {
 	return err
 }
 
+func (c *checker) checkForOldJndiManagerConstructor(filename string) bool {
+	return !c.hasOldJndiManagerConstructor && strings.Contains(filename, "JndiManager")
+}
+
+func (c *checker) checkJndiManagerVersion(filename string) bool {
```

Comment:
it's weird to have a function that takes a receiver and doesn't use it.

Reply:
Fixed.